SC_redesign/Crop_and_Soil/soil/layer_data.py

```diff
@@ -17,36 +18,59 @@ class LayerData:
     """nitrate level of the layer (kg/ha)"""
     soil_water_concentration: float = 0.25  # arbitrary
     """soil water concentration of the layer (mm)"""
+    soil_water_content:  Optional[float] = None  
+    """volume of soil water in the layer (mm)"""
     field_capacity_water_concentration: float = 0.3  # arbitrary
     """water concentration of soil layer at field capacity (mm)"""
     wilting_point_water_concentration: float = 0.2  # arbitrary
     """water concentration of soil layer at wilting point (mm)"""
     saturation_point_water_concentration: float = 0.5
     """water concentration of soil layer at saturation point (mm)"""
```

Comment:
Just to make this a bit clearer, since water is measured in a confusing unit:

```suggestion
    field_capacity_water_concentration: float = 0.3  # arbitrary
    """water concentration of soil layer at field capacity (mm water / mm soil)"""
    wilting_point_water_concentration: float = 0.2  # arbitrary
    """water concentration of soil layer at wilting point (mm water / mm soil)"""
    saturation_point_water_concentration: float = 0.5
    """water concentration of soil layer at saturation point (mm water / mm soil)"""
```

Reply:
Whatever we can do to make the water units less confusing, I am all for it